dashboard/app/controllers/plc/course_units_controller.rb

```diff
@@ -0,0 +1,76 @@
+class Plc::CourseUnitsController < ApplicationController
+  load_and_authorize_resource
+
+  # GET /plc/course_units
+  # GET /plc/course_units.json
+  def index
+  end
+
+  # GET /plc/course_units/1
+  # GET /plc/course_units/1.json
+  def show
+  end
+
+  # GET /plc/course_units/new
+  def new
+    @course_unit.plc_course_id = params[:plc_course_id]
+  end
+
+  # GET /plc/course_units/1/edit
+  def edit
+  end
+
+  # POST /plc/course_units
+  # POST /plc/course_units.json
+  def create
+    if @course_unit.save
+      redirect_to @course_unit, notice: 'Course unit was successfully created.'
+    else
+      redirect_to action: :new
+    end
+  end
+
+  # PATCH/PUT /plc/course_units/1
+  # PATCH/PUT /plc/course_units/1.json
+  def update
+    if @course_unit.update(course_unit_params)
+      redirect_to @course_unit, notice: 'Course unit was successfully created.'
+    else
+      redirect_to action: :edit
+    end
+  end
+
+  # DELETE /plc/course_units/1
+  # DELETE /plc/course_units/1.json
+  def destroy
+    @course_unit.destroy
+    redirect_to plc_content_creator_show_courses_and_modules_path
+  end
+
```

Comment:
Since questions and answers are associated with course units instead of courses, the logic for submitting new questions and answers needs to move to the Course Unit controller